src/js/common/schemaform/FormPage.jsx

```diff
@@ -77,11 +54,10 @@ class FormPage extends React.Component {
       newData = _.set([this.props.route.pageConfig.arrayPath, this.props.params.index], formData, this.props.form.data);
     }
     this.props.setData(newData);
-    this.debouncedAutoSave();
   }
 
   onSubmit({ formData }) {
```

Comment:
Just making sure this is intentionally not `onSubmit = ({ formData }) => {...}` after you got rid of `this.onSubmit = this.onSubmit.bind(this)`.

Reply:
Nope, not intentional. Good catch.